core/server/services/auth/members/index.js

```diff
@@ -1,27 +1,28 @@
-const jwt = require('jsonwebtoken');
```

Comment:
This commit will be much easier to review with `?w=1` and the end of the url :+1: